pkg/backend/redis/queue.go

```diff
@@ -218,7 +218,7 @@ func (b *Backend) DeleteQueue(ctx context.Context, queueName string) error {
 	// .. task_keys = collect task keys
 	// WATCh task_keys
 	// MULTI
-	// DEL {queue_key} worker_keys task_keys
+	// UNLINK {queue_key} worker_keys task_keys
```

Comment:
could you comment this is performed in chunked manner?

```suggestion
	// UNLINK {queue_key} worker_keys task_keys (chunked)
```

Reply:
Thank you, I should have commented.